src/FileService.Data.Provider.MsSql.EF/Migrations/20210416201900_FixImageTable.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LT.DigitalOffice.FileService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.FileService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(FileServiceDbContext))]
+    [Migration("20210416201900_FixImageTable")]
+    public class FixImageTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: nameof(DbImage.ImageType),
+                table: DbImage.TableName);
+
+            migrationBuilder.AddColumn<int>(
+               name: nameof(DbImage.ImageType),
+               table: DbImage.TableName,
+               nullable: false);
```

Comment:
nameof is not very good. If the property name is renamed, it may stop working if somewhere higher or lower in migrations there is string "ImageType"

Reply:
if property name is renamed, we should be change property in nameof